test/integration/configuration.test.js

```diff
@@ -69,6 +69,14 @@ describe(Support.getTestDialectTeaser('Configuration'), () => {
     });
   });
 
+  describe('Instantiation with a URL string', () => { 
```

Comment:
You have added this test to incorrect file, I asked for `unit` not `integration` folder

Reply:
Ops, sorry my bad, fixed